src/tests/database.spec.ts

```diff
@@ -342,12 +341,7 @@ describe("#rdsInstanceBackupEnabled", () => {
     });
 
     describe("do not check read replicas", () => {
-        const policy = database.rdsInstanceBackupEnabled({
-            enforcementLevel: "mandatory",
-            backupRetentionPeriod: 7,
-            preferredBackupWindow: "window",
-            checkReadReplicas: false,
```

Comment:
Should these be config values passed as the new `config` arg to `createResourceValidationArgs` below?

Reply:
I'm not totally sure where you mean?  I will push up latest commits and perhaps you could take another look?  Thank you!